bin/dm_mripicture.py

```diff
@@ -14,11 +14,16 @@
     -s --subject        Subjects
     -o --output=FOLDER  Output directory
     -t --tag=TAG        Scan tag [default: T1]
+    -r --replace        Replace existing files [default: False]
```

Comment:
Suggesting changing this to `--force` to align with other typical command line interfaces.

```suggestion
    -f --force        Force overwrite of output files [default: False]
```